mapper/dm_agent/README.md

```diff
@@ -0,0 +1,31 @@
+# Device Monitoring
+
+Device monitoring is enabled on `thin-edge.io` using `collectd` and the `tedge-dm-agent` component.
+
+# Prerequisites
+
+Install collectd if not already installed: 
+
+`sudo apt install collectd`
+
+Configure collectd. Use the Thin Edge customized collectd configuration for basic monitoring: 
+
+`sudo cp $TEDGE_REPO/configuration/contrib/collectd/collectd.conf /etc/collectd/collectd.conf`
```

Comment:
What is `$TEDGE_REPO`?

Reply:
File name is wrong, it should be configuration/contrib/collectd/tedge-collectd.conf

Reply:
I've renamed that config file from `tedge-collectd.conf` to just `collectd.conf` in this PR itself.